website/docs/r/app_configuration.html.markdown

```diff
@@ -192,6 +199,14 @@ An `identity` block supports the following:
 
 ---
 
+A `replica` block supports the following:
+
+* `location` - (Required) Specifies the supported Azure location where the replica exists.
```

Comment:
Changing this will also force a new replica to be created so can we add that line in here as well?

Reply:
fixed, thanks.